packages/opal-server/opal_server/server.py

```diff
@@ -225,6 +226,9 @@ def _configure_api_routes(self, app: FastAPI):
         def healthcheck():
             return {"status": "ok"}
 
+        # new API
+        app.mount("/api", build_api())
```

Comment:
Maybe write a better name? Or add a parameter of the version to be added (for future use)